include/glow/Backends/BackendUtils.h

```diff
@@ -30,6 +30,10 @@ struct RuntimeSymbolInfo {
   size_t offset;
   /// Type of symbol.
   Type type;
+  /// Is the symbol an input for the function.
```

Comment:
please run "format.sh fix"

Reply:
Format.sh fix didn't change anything? Is it the indentation?

Reply:
Yes, I would like the indentation should be the consistent with the rest of file.